### PR TITLE
Iron chunks normals

### PR DESCRIPTION
**Pull request by Nunez2196**
**Brief Description of What This PR Does**

Changes the Iron Chunk shader to use normal mapping along with METALLIC and ROUGHNESS.

**Related Issues**

no related issue, just noticed unused files

**Progress Checklist**

Note: before starting this checklist the issue should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed
- [x] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author.

### CODE REVIEW

File: shaders/IronChunk.shader [hunk 1]
```diff
@@ -1,7 +1,8 @@
 shader_type spatial;
 render_mode depth_draw_alpha_prepass;
 
-uniform sampler2D texture : hint_albedo;
+uniform sampler2D Texture : hint_albedo;
+uniform sampler2D Normals;
```
Comment (hhyyrylainen) [inline shaders/IronChunk.shader:5]: these don't follow the naming convention in other shaders

Comment (Nunez2196) [reply shaders/IronChunk.shader:5]: will change it right now

Comment (Nunez2196) [reply shaders/IronChunk.shader]: hold on, godot reimported something messed up

File: shaders/IronChunk.shader [hunk 2]
```diff
@@ -10,7 +11,8 @@ uniform float outlineWidth;
 uniform vec4 growColor : hint_color;
 
 void fragment() {
-    vec4 mainTex = texture(texture, UV);
+    vec4 mainTex = texture(albedo, UV);
+    vec4 normalmap = texture(normals, UV);
```
Comment (Kasterisk) [inline shaders/IronChunk.shader:15]: camelCase:
```suggestion
    vec4 normalMap = texture(normals, UV);
```

**Comment by hhyyrylainen** [comment]
@Nunez2196 is this PR ready for review? You haven't checked the first checkbox in the PR description.

**Comment by Nunez2196** [comment]
> @Nunez2196 is this PR ready for review? You haven't checked the first checkbox in the PR description.

yes, i'm terribly sorry, forgot about the checklist. I fixed the things you guys pointed out and @buckly90 also tested this out and it worked.

**Review by hhyyrylainen — APPROVED** [review]

**Pull request merged**